README.md

```diff
@@ -22,6 +23,17 @@ Welcome to Bridge-To-Kubernetes! Bridge to Kubernetes extends the Kubernetes per
 - [Visual Studio](https://docs.microsoft.com/en-us/visualstudio/bridge/bridge-to-kubernetes-vs)
 - [Visual Studio Code](https://docs.microsoft.com/en-us/visualstudio/bridge/bridge-to-kubernetes-vs-codeart)
 
+## CLI tool installation
+- ```curl -fsSL https://raw.githubusercontent.com/Azure/Bridge-To-Kubernetes/main/scripts/install.sh | bash```
+- Supports Linux, Darwin, Windows - use WSL (installation link [here](https://learn.microsoft.com/en-us/windows/wsl/install)) or Git Bash (installation link [here](https://git-scm.com/))
+
+## How to use the CLI
+-  cd into the installation folder for linux or WSL ex: ``` $HOME/.local/bin/bridgetokubernetes``` for darwin cd into installation folder here ```/usr/local/bin/bridgetokubernetes```

```

Comment:
@sabbour This is because similar tools like this are installed in local/bin folder so that it can be accessed globally. Most of the tools don't specify them in readme, ex: look [here](https://github.com/metalbear-co/mirrord#cli-tool). Even draft installation happens at same folder structure. I can remove them if it creates confusion. Please let me know. Thank you.

Reply:
If installing them in /local/bin would make them globally accessible, why would someone need to change directory into /local/bin to run it?

Reply:
I will update them; it is valid point.